### ISSUE TITLE
App crashes when opening video or song

### ISSUE THREAD

**Issue by devz3r0** [nextcloud/android#2877]
### Actual behaviour
- Tell us what happens
App stuck after downloading a song or video and trying to process it. 
### Expected behaviour
- Tell us what should happen
 Open the song or video or tell that it is unable to play it back. 

### Steps to reproduce
1. Install app
2. Connect to nextcloud
3. Open a media file. 

### Environment data
Android version:
8.1 lineage
Device model: 
One plus 2
Stock or customized system:
Custom. 
Nextcloud app version:
3.2.1
Nextcloud server version:
13.0.5

**Comment by AndyScherzinger** [comment]
@tobiasKaminsky can you reproduce this issue since you also have a OP2?

@devz3r0 can you tell us what file type leads to this issue?

**Comment by devz3r0** [comment]
@AndyScherzinger it happens with every media file, f.E. .mp4 .mp3 .ogg etc.
If i open that downloaded file with "open with" with an external program, that works.

**Comment by tobiasKaminsky** [comment]
I have a OnePlus 3 with lineageOS and it is working without any problem.
I tried mp3 and video.

Can you provide us additional infos via logcat? https://github.com/nextcloud/android/blob/master/README.md#getting-debug-info-via-logcat

**Comment by devz3r0** [comment]
is that enough?

build.board: MSM8994
build.bootloader: unknown
build.brand: OnePlus
build.cpu_abi: arm64-v8a
build.cpu_abi2:
build.device: OnePlus2
build.display: lineage_oneplus2-userdebug 8.1.0 OPM2.171019.029.B1 9acd941d5c
build.fingerprint: OnePlus/OnePlus2/OnePlus2:6.0.1/MMB29M/1447858500:user/release-keys
build.hardware: qcom
build.host: agrippa.acc.umu.se
build.id: OPM2.171019.029.B1
build.manufacturer: OnePlus
build.model: ONE A2003
build.product: OnePlus2
build.radio: unknown
build.serial: 48c4f866
build.tags: release-keys
build.time: 1526930510000
build.type: user
build.user: jenkins
version.codename: REL
version.incremental: 9acd941d5c
version.release: 8.1.0
version.sdk_int: 27

08-15 10:46:01.950 I/InputDispatcher(946): Application is not responding: Window{e39be4e u0 com.nextcloud.client/com.owncloud.android.ui.activity.FileDisplayActivity}. It has been 5008.4ms since event, 5006.4ms since wait started. Reason: Waiting to send non-key event because the touched window has not finished processing certain input events that were delivered to it over 500.0ms ago. Wait queue length: 3. Wait queue head age: 6068.7ms.
08-15 10:46:01.970 I/WindowManager(946): Input event dispatching timed out sending to com.nextcloud.client/com.owncloud.android.ui.activity.FileDisplayActivity. Reason: Waiting to send non-key event because the touched window has not finished processing certain input events that were delivered to it over 500.0ms ago. Wait queue length: 3. Wait queue head age: 6068.7ms.
08-15 10:46:05.949 E/ActivityManager(946): ANR in com.nextcloud.client (com.nextcloud.client/com.owncloud.android.ui.activity.FileDisplayActivity)
08-15 10:46:05.949 E/ActivityManager(946): 3.5% 22309/com.nextcloud.client: 2.8% user + 0.6% kernel / faults: 11491 minor
08-15 10:46:14.627 W/ActivityManager(946): Force finishing activity com.nextcloud.client/com.owncloud.android.ui.activity.FileDisplayActivity
08-15 10:46:14.650 I/ActivityManager(946): Killing 22309:com.nextcloud.client/u0a136 (adj 0): user request after error
08-15 10:46:14.747 W/InputDispatcher(946): channel 'e39be4e com.nextcloud.client/com.owncloud.android.ui.activity.FileDisplayActivity (server)' ~ Consumer closed input channel or an error occurred. events=0xd
08-15 10:46:14.748 E/InputDispatcher(946): channel 'e39be4e com.nextcloud.client/com.owncloud.android.ui.activity.FileDisplayActivity (server)' ~ Channel is unrecoverably broken and will be disposed!
08-15 10:46:14.749 I/WindowManager(946): WIN DEATH: Window{e39be4e u0 com.nextcloud.client/com.owncloud.android.ui.activity.FileDisplayActivity}
08-15 10:46:14.750 W/InputDispatcher(946): Attempted to unregister already unregistered input channel 'e39be4e com.nextcloud.client/com.owncloud.android.ui.activity.FileDisplayActivity (server)'

**Comment by devz3r0** [comment]
actually the "crash" will not result in closing the app, it just stops doing anything. Then you get the ANR message box.

**Comment by tobiasKaminsky** [comment]
Is this also happening when you first download the file via three dots menu and then click on it?

**Comment by devz3r0** [comment]
no, it only happens if the internal media player of the nextcloud app want to handle the file.
It does not matter if you need to download it first, or not. If you need to download it, it does that then get stuck. However the downloaded file has finished downloading and is available to the system.
